src/subscription/index.scss

```diff
@@ -64,6 +64,9 @@
     }
   }
 
+  .monthly-legal-notification p.micro {
+    font-size: 12px !important; 
```

Comment:
Not sure, should we prefer `rem` here over `px`?

Reply:
We need to make it specific to 'px'  
Because it's a parallel text to stripe legal text and we want to make it look exactly same.

Reply:
I see, thanks for the clarification